worldeditadditions_commands/commands/selectors/scloud.lua

```diff
@@ -0,0 +1,35 @@
+-- ███████  ██████ ██       ██████  ██    ██ ██████
+-- ██      ██      ██      ██    ██ ██    ██ ██   ██
+-- ███████ ██      ██      ██    ██ ██    ██ ██   ██
+--      ██ ██      ██      ██    ██ ██    ██ ██   ██
+-- ███████  ██████ ███████  ██████   ██████  ██████
+local wea = worldeditadditions
+worldedit.register_command("scloud", {
+	params = "0-6/stop/reset",
```

Comment:
This would be more consistent:
```suggestion
	params = "<0-6|stop|reset>",
```

Reply:
world edit did it as
```
worldedit.register_command("p", {
	params = "set/set1/set2/get",
```
But if you really want it changed I'll commit this.

Reply:
I see. I would if you don't mind, as it's more consistent with the rest of WorldEditAdditions (I use this syntax elsewhere).